packages/falso/src/lib/text.ts

```diff
@@ -1,6 +1,67 @@
-import { FakeOptions, fake } from './core';
-import { data } from './text.json';
+import { FakeOptions, fake, getRandomInRange } from './core';
+import { word } from './word';
 
-export function text<Options extends FakeOptions>(options?: Options) {
-  return fake(data, options);
+export interface TextOptions extends FakeOptions {
+  charCount?: number;
+}
+
+function getSpecialCharacter(): string {
+  const randomNumber = getRandomInRange({ min: 1, max: 10, fraction: 0 });
+
+  if (randomNumber === 1) {
+    return '.';
+  }
+
+  if (randomNumber === 2) {
+    return ',';
+  }
+
+  return '';
+}
+
+function generateSentence(charCount: number): string {
```

Comment:
Please use it in `sentence.ts `

Reply:
That's a good point. What is the difference between sentence and text? Is there a need for both?

If I move my sentence building logic into sentence.ts, what will be the difference between the 2?

Reply:
I think that a sentence should be one sentence and the text is for a bigger chunk of text.

Reply:
Ok yeah, that makes sense 😊

Just to clarify, do you want me to:
1. Replace the current `sentence()` logic with my `generateSentence` logic (tweaking it slightly so that it can generate single sentences). Then call `sentence()` in `text.ts` to make up the `charCount` for `text()`
2. Delete the generateSentence logic. Use the current `sentence` functionality to make up the `charCount` for `text()`?

Reply:
Ok cool. I will get that done after work today.